### ISSUE TITLE
Is the author still there

### ISSUE THREAD

**Issue by q1165600785** [pmlmodelling/pyfvcom#7]
Excuse me, boss. Is this library no longer under maintenance? Are you no longer engaged in research on FVCOM

**Comment by marius-dewar** [comment]
Hi there, the library is in the process of being updated. I'm sorry for the delay, we have minimal time to work on coding.

**Comment by q1165600785** [comment]
@marius-dewar  

I really appreciate your maintenance and contribution. I tried to modify the code myself.

I remember that the API of the proj library has changed. There are also changes in the API of the scipy and numpy libraries. I have tried to modify them, but there have been problems with the high-level water level of utide calculation of the boundary points. I consulted the maintainer of utide.
https://github.com/wesleybowman/UTide/issues/134
In the end, the problem was not solved. I don’t know where the problem is.

**Comment by marius-dewar** [comment]
Yeah we have been looking at these issues here and resolved the scypi and numpy issues at least for the preprocessing. The high tide elevation caused a lot of confusion here too, turns out utide had changed it's default baseline date to 0000-00-00 so we need to change this to mjd when calling utide

**Comment by marius-dewar** [comment]
Try the pushes I added yesterday to the dev branch and let me know if it fixes things

**Comment by q1165600785** [comment]
@marius-dewar   I will try it as soon as possible, and feel your maintenance again；

I have two more questions. I use oceanmesh (https://github.com/CHLNDDEV/oceanmesh) to generate triangulated meshes. I use SMS to open the generated 2dm file. I find that it distinguishes land boundaries from ocean boundaries. However, I read the pyfvcom source code, and it sets all NS nodes as ocean boundaries. I find that fort14 format cannot distinguish between ocean boundaries and land boundaries. Only grid file format can distinguish between ocean boundaries and land boundaries.

There is another question. I use pyfvcom to generate fvcom configuration files. I find that the configuration has changed. This also needs to be modified.

I appreciate your maintenance again. I will use it as soon as possible.

**Comment by marius-dewar** [comment]
For these you could make a branch and modify it as you need as there are so many versions of FVCOM that are still in use that the configuration will never match. We only mark out the open boundary as it saves time, but again feel free to modify the code to match your own needs.

**Comment by q1165600785** [comment]
@marius-dewar   Okay, I will pull a branch for code writing. Can my code be merged into the mainstream branch，I will try to write my own code to implement my ideas

**Comment by marius-dewar** [comment]
Yes as it's a community development we try to merge all developments and bug fixes from other branches, make sure you put a comment on the code you change with your name if you would like credit for the work too.
